templates/search_page.html

```diff
@@ -139,7 +140,29 @@ <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
                     </div>
 
                 </div>
-
+                <div class="px-1 mt-1" style="width: 6em;">
+                    <select name="price" class="form-select border border-secondary"
+                        aria-label=".form-select-sm example" id="priceToSelect" onchange="getRandomInt(this)">
+                        <option>price</option>
+                        <option value="0">$</option>
+                        <option value="1">$$</option>
+                        <option value="2">$$$</option>
+                        <option value="3">$$$$</option>
+                        <option value="4">Max</option>
+                        <option value="0,1,2,3,4">Surprise</option>
+                        <script>
```

Comment:
How about putting JS logic codes (line 154 -- line 162) into `search.js`, because this HTML cites `search.js` anyway? So we can have a better organization of difference pieces of codes, html, css and JS etc.

Reply:
@sananand007 this is the comment that I shared earlier.